__init__.py

```diff
@@ -1 +1,2 @@
 from . import models
+from . import wizards
```

Comment:
To maintain the Odoo standard I think the folder should be called "wizard".

```suggestion
from . import wizard
```

Reply:
Quite the opposite. Plural, please, to conform OCA guidelines

Reply:
Sorry for re confusion.

Reply:
I am following the file naming of the OCA -> https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming
![temp](https://user-images.githubusercontent.com/8931673/168479560-1b525423-1a81-489e-bf1c-17997e378f56.png)
.